fetch.bs

```diff
@@ -1121,6 +1109,46 @@ or "<code>worker</code>".
  <li><p>Return <var>newRequest</var>.
 </ol>
 
+<p>To <dfn export for=request id=concept-request-upload-body>upload body</dfn> of a
```

Comment:
Then which term should we use? In other words, what are the "primitive" operations on the network layout which I can use?